packages/block-library/src/file/style.scss

```diff
@@ -2,10 +2,6 @@
 	// This block has customizable padding, border-box makes that more predictable.
 	box-sizing: border-box;
 
-	&:not(.wp-element-button) {
-		font-size: 0.8em;
-	}
-
```

Comment:
Won't this break back compat for themes that may be relying on this CSS?

Reply:
Yes, but the style was not originally there: the addition of the style was a back compat break itself.
In 2022, it was moved from the button to the link text. That is the origin of the open issue.